packages/odyssey-react/src/components/Table/TableDataCell.tsx

```diff
@@ -37,7 +38,7 @@ export const TableDataCell = withTheme(
     const componentClass = useCx(
       styles.root,
       format && styles[`${format}Format`],
-      empty && styles[`${empty}State`]
+      empty && styles[`emptyState`]
```

Comment:
This previously resulted in the component looking for `.trueState` in the module.

Reply:
@edburyenegren-okta hmmmm maybe its time to explore the type supported CSS modules idea I floated a few months back, makes bugs like this go away entirely and we don't have full unit test coverage over the application of classes although I think we might want to consider it!

Reply:
I like this idea, but I'd rather handle it outside of this PR, so the migration isn't hung up by any exploration we'll need to do there.

Reply:
@edburyenegren-okta yup of course, I was thinking aloud but not something we need in this PR